### PR TITLE
Update gravatar as a default avatar

### PR DESCRIPTION
**Pull request by hendri-tobing**
When registering as a new user, it is automatically getting a default avatar from https://www.gravatar.com as described in #571 . If the new user does not have it on gravatar, the default avatar will be the vutuv logo.

### CODE REVIEW

File: lib/vutuv/accounts.ex [hunk 1]
```diff
@@ -282,6 +286,48 @@ defmodule Vutuv.Accounts do
...
+      |> :erlang.md5()
+      |> Base.encode16(case: :lower)
+
+    Application.ensure_all_started(:inets)
+    Application.ensure_all_started(:ssl)
+
+    img = 'https://www.gravatar.com/avatar/#{hash}?s=150&d=404'
+
+    if {:ok, {status, header, body}} = :httpc.request(:get, {img, []}, [], []) do
+      case status do
```
Comment (riverrun) [inline lib/vutuv/accounts.ex:309]: This case can be simplified - for example:

```
case status do
  {_, 200, 'OK'} -> create_upload
  _ -> nil
end
```

File: lib/vutuv/accounts.ex [hunk 2]
```diff
@@ -282,6 +286,48 @@ defmodule Vutuv.Accounts do
...
+          nil
+
+        {_, 200, 'OK'} ->
+          content_type = find_content_type(header) |> to_string
+          filextension = String.replace(content_type, "image/", "")
+          filename = "original.#{filextension}"
+          path = "#{Application.get_env(:vutuv, :storage_dir)}" <> "#{userid}/"
+          File.write(path <> filename, body)
+
+          _upload = %{
```
Comment (riverrun) [inline lib/vutuv/accounts.ex]: We don't need `_upload = ` here - just return the map.

File: lib/vutuv/accounts.ex [hunk 3]
```diff
@@ -80,7 +82,9 @@ defmodule Vutuv.Accounts do
 
     case Repo.insert(user_with_email) do
       {:ok, new_user} ->
-        user_with_profile = Ecto.build_assoc(new_user, :profile, profile_attrs)
+        img = get_gravatar(attrs["email"], new_user.id)
```
Comment (riverrun) [inline lib/vutuv/accounts.ex:85]: What happend when this function returns nil?

**Review by riverrun — CHANGES_REQUESTED** [review]
Looks good - just a couple of small comments and a question.

File: lib/vutuv/accounts.ex (hunk 3 above)
Comment (hendri-tobing) [reply lib/vutuv/accounts.ex:85]: If the get_gravatar return nil, the avatar will be the default avatar of Vutuv (located at assets/static/images/avatars/)

**Comment by hendri-tobing** [comment]
I have updated as advised. Please let me know if anything is missing. Thank you.

**Review by riverrun — APPROVED** [review]

**Pull request merged**